Tiltfile

```diff
@@ -1,6 +1,10 @@
 # -*- mode: Python -*-
 
-load("ext://uibutton", "cmd_button", "text_input")
+envsubst_cmd = "./hack/tools/bin/envsubst"
+clusterctl_cmd = "./bin/clusterctl"
+kubectl_cmd = "kubectl"
```

Comment:
Let's please verify that those binaries exist, and for the first two let's give a hint in the error message how to build them

Reply:
For the first two: what about running `make envsubst clusterctl` via tilt to ensure they are there?

Disclamer: I'm bad at tilt, it may be a bad idea :-)

Reply:
I think it should be fine to just run the targets

Reply:
@sbueringer Just to clarify, how do we want to handle the binaries here? Do we want to check that each command exists in the beginning of the Tiltfile, or instead have the Makefile generate all of them if they're not there? And in the latter case, is there a way to have "tilt up" trigger that generation?

Reply:
I would add something like this (we had something similar it in the past for other binaries)
```
def ensure_envsubst():
    local("make {}".format(envsubst_cmd))
```

And then call it in ~ l. 394.

This will always call the make target and the make target can then figure out if it has to built the binary or not

Reply:
For kubectl we can only check it exists and fail otherwise